catalog/catalog-app/pom.xml

```diff
@@ -511,5 +511,21 @@
             <artifactId>catalog-core-downloadaction</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>ddf.catalog.plugin</groupId>
+            <artifactId>catalog-plugin-clientinfo</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>ddf.platform</groupId>
+            <artifactId>platform-filter-clientinfo</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <!--Is this even necessary?-->
```

Comment:
You should determine the answer to the question and then delete the comment or the dependency accordingly.